app/src/main/java/com/firebase/uidemo/RecyclerViewDemoActivity.java

```diff
@@ -201,40 +201,29 @@ public String getText() {
     }
 
     public static class ChatHolder extends RecyclerView.ViewHolder {
-        View mView;
+        public View mView;
```

Comment:
If you want an explicit access qualifier, I don't want it to be `public`. Make it `private` or `protected`.